jsettlers.graphics/src/main/java/jsettlers/graphics/map/controls/original/panel/content/settlers/profession/ProfessionPanel.java

```diff
@@ -155,18 +151,14 @@ public SettlerPanel(float width, EProfessionType type) {
 
 				add(Panel.box(new CountArrows(() -> {
 					if (getTotalRatio() <= 1.0f - 0.05f) {
-						setRatio(Math.min(1f, ratio + 0.05f));
-						lastChangeTimestamp = System.currentTimeMillis();
-						SettlerPanel.this.update();
+						this.ratio = Math.min(1f, ratio + 0.05f);
 						return new SetMoveableRatioAction(type.moveableType, position, ratio);
```

Comment:
This still has the issue that the new value based of the saved value.
If you double click on one arrow you can see in the console that two events are send but they both contain the same value.
This is why I said that you shouldn't send the ratio but the delta.
for example:
new ChangeMovableRatioAction(EActionType.INCREASE_MOVABLE_RATIO, type.movableType, position)